src/currencyservice/package.json

```diff
@@ -10,9 +10,14 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@google-cloud/debug-agent": "3.2.0",
+    "@google-cloud/opentelemetry-cloud-trace-exporter": "^0.4.1",
     "@google-cloud/profiler": "4.0.2",
-    "@google-cloud/trace-agent": "2.11.0",
+    "@google-cloud/trace-agent": "5.1.0",
```

Comment:
is this needed now that we use opentelemetry tracing?

Reply:
:'((((( yeah i tried to delete it... and suddenly thousands of `grpc.google.devtools.cloudtrace.v2.TraceService/BatchWriteSpans` traces were coming out of nowhere, which caused the quota exceeded and probably the CI error. still trying to figure out what's going on, so i'm looking to see what variables i can turn on/off

Reply:
alright it's GONE